rotary/rules/rotary.smk

```diff
@@ -314,17 +314,17 @@ rule qc_long_length_stats:
 
 rule qc_long:
     input:
-        expand("{sample}/stats/qc_long_length_stats.txt",sample=SAMPLE_NAMES)
+        expand("{sample}/stats/{sample}_qc_long_length_stats.txt",sample=SAMPLE_NAMES)
     output:
         temp(touch("checkpoints/qc_long"))
 
 
 rule assembly_flye:
     input:
-        "{sample}/qc_long/nanopore_qc.fastq.gz"
+        "{sample}/qc_long/{sample}_nanopore_qc.fastq.gz"
     output:
-        assembly="{sample}/assembly/flye/assembly.fasta",
-        info="{sample}/assembly/flye/assembly_info.txt",
+        assembly="{sample}/assembly/flye/{sample}_assembly.fasta",
```

Comment:
Flye does not allow you to specify the names of the output files in its output folder. So perhaps we should leave `assembly.fasta` and `assembly_info.txt` as-is? Otherwise, we will need to add `mv` commands to the shell section here.

Reply:
(This comment also applied to line 327 below)

Reply:
@jmtsuji Thanks! Added mv commands as part of my debugging efforts.